htdocs/core/tpl/list_print_total.tpl.php

```diff
@@ -59,4 +35,87 @@
 		}
 	}
 	print '</tr>';
+	// Add grand total if necessary ie only if different of page total already printed above
+	if (getDolGlobalString('MAIN_GRANDTOTAL_LIST_SHOW') && (!(is_null($limit) || $num < $limit))) {
+		if (isset($totalarray['pos']) && is_array($totalarray['pos']) && count($totalarray['pos']) > 0) {
+			$sumsarray = false;
+			$tbsumfields = [];
+			foreach ($totalarray['pos'] as $field) {
+				$tbsumfields[] = "sum($field) as `$field`";
```

Comment:
Use of backtick ` is forbidden in SQL as it is not standard.
May be you can do:
```
$fieldforsum = preg_replace('/[^a-z0-9]/', '', $field);
$tbsumfields[] = "sum($field) as $fieldforsum";
```